tools/cldr-apps/src/main/java/org/unicode/cldr/web/VettingViewerQueue.java

```diff
@@ -351,6 +281,87 @@ public void done() {
             }
         }
 
+        private void processCriticalWork(final CLDRProgressTask progress) {
+            VettingViewer<Organization> vv;
+            status = "Beginning Process, Calculating";
+
+            vv = new VettingViewer<>(sm.getSupplementalDataInfo(), sm.getSTFactory(),
+                getUsersChoice(sm), "Winning " + SurveyMain.getNewVersion());
+            progress.update("Got VettingViewer");
+            statusCode = Status.PROCESSING;
+            start = System.currentTimeMillis();
+            last = start;
+            n = 0;
+            vv.setProgressCallback(new VettingViewer.ProgressCallback() {
+                public String setRemStr(long now) {
```

Comment:
processCriticalWork is too long and deeply indented; how about moving setProgressCallback to a subroutine

Reply:
actually the anonymous class is what needs to have a name.